src/game/clients/CClientLog.cpp

```diff
@@ -222,8 +222,8 @@ bool CClient::addRelay( const CServerDef * pServ )
 		CSString sCustomerID(pServ->GetName());
 		sCustomerID.Add(GetAccount()->GetName());
 
-		dwCustomerId = z_crc32(0L, Z_NULL, 0);
-		dwCustomerId = z_crc32(dwCustomerId, reinterpret_cast<const z_Bytef *>(sCustomerID.GetBuffer()), (z_uInt)sCustomerID.GetLength());
+		dwCustomerId = crc32(0L, Z_NULL, 0);

```

Comment:
Was it necessary to remove the z_ prefix? Afair, we have a zlib config header with the Z_PREFIX macro set to 1, so i can't think why that's necessary

Reply:
https://github.com/Sphereserver/Source-X/blob/master/src/cmake/toolchains/Linux-GNU-64.cmake#L83 yes i have see it here, but for some reason the compiler dont get it let me check that again